app/src/main/java/com/realityexpander/tasky/agenda_feature/domain/IRemindAtAlarmManager.kt

```diff
@@ -0,0 +1,16 @@
+package com.realityexpander.tasky.agenda_feature.domain
+
+import android.content.Context
+
+interface IRemindAtAlarmManager {
+
+    fun setAlarmsForAgendaItems(
+        context: Context,
+        agendaItems: List<AgendaItem>
+    )
+
+    fun cancelAllAlarms(
+        context: Context,
+        onFinished: () -> Unit = {},
+    )
```

Comment:
Instead of providing the context as parameter in the domain layer (which is Android-specific and therefore not isolated), I'd pass it to the constructor of the class implementing `IRemindAtAlarmManager`.
Otherwise, you can't create fakes of this interface for local JVM tests.

Reply:
OK, i will refactor to use hilt